src/platform-includes/performance/add-active-span/javascript.mdx

```diff
@@ -0,0 +1,29 @@
+You can use the `Sentry.startActiveSpan` method to wrap a callback in a span to measure how long it will take. The span is automatically finished when the callback is finished. This will work with both sync and async callbacks.
+
+```javascript
+const result = Sentry.startActiveSpan({ name: "Important Function" }, () => {
+  return expensiveFunction();
+});
+
+const result = await Sentry.startActiveSpan(
+  { name: "Important Function" },
+  async () => {
+    const res = Sentry.startActiveSpan({ name: "Child Span" }, () => {
+      return expensiveFunction();
+    });
+
+    return updateRes(res);
+  }
+);
+
+const result = Sentry.startActiveSpan(
+  { name: "Important Function" },
+  (span) => {
+    // Can access the span to add data or set specific status
+    span?.setData("foo", "bar");
```

Comment:
m: Should we mention/explain when/why `span` is undefined? May not be entirely clear from the outside why this may be unset and why I need to guard here.

Reply:
yup, ill add a note